site/src/pages/release-notes/1.20.1.mdx

```diff
@@ -0,0 +1,41 @@
+---
+date: 2022-10-11
+---
+
+## 🌟 New features
+
+- N/A
+
+## 📈 Improvements
+
+- N/A
+
+## 🛠️ Bug fixes
+
+- `application/json` media type is correctly set to HTTP/JSON-to-gRPC transcoding responses. #4474
```

Comment:
What do you think of also documenting the affected versions?

```suggestion
- `application/json` media type is correctly set for HTTP/JSON-to-gRPC transcoded responses. #4474
    - Versions 1.19.0 to 1.20.0 are affected
```

Reply:
; and link `#4473` as well?

Reply:
Fixed. 😉